### PR TITLE
[RFR][1LP] New test test_embed_tower_playbooks_list_changed

### PR DESCRIPTION
**Pull request by quarckster**
Intent
=================

__Adding test__ test_embed_tower_playbooks_list_changed. It tests if playbooks list changed after playbooks repo removing.

{{pytest: -v -k "test_embed_tower_playbooks_list_changed" --long-running}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit ca3d5e0080d4dc344076cd20e2967c0904ee5312:

`cfme/ansible/playbooks.py`:
- :red_circle: Line 4:1: **F401** *'widgetastic.widget.ConditionalSwitchableView' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 1bcef612c2fa3c808fe93c319ea6993a254df176:

`cfme/ansible/playbooks.py`:
- :red_circle: Line 5:1: **F401** *'widgetastic_manageiq.Table' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 16d34f950ca5081979d3348f8677b2d7bfabe90c:

`cfme/tests/ansible/test_embedded_ansible_basic.py`:
- :red_circle: Line 8:1: **F401** *'cfme.ansible.playbooks.PlaybooksCollection' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/ansible/repositories.py [hunk 1]
```diff
@@ -277,6 +291,10 @@ def refresh(self):
         view.flash.assert_no_error()
         view.flash.assert_message("Embedded Ansible refresh has been successfully initiated")
 
+    @property
```
Comment (mfalesni) [inline cfme/ansible/repositories.py]: OPTIONAL: You can probably use cached_property here. I use it in automate as well.

**Pull request merged**